packages/react/src/components/ComboBox/ComboBox.tsx

```diff
@@ -134,17 +309,17 @@ const ComboBox = React.forwardRef((props, ref) => {
   }
 
   const filterItems = (items, itemToString, inputValue) =>
-    items.filter((item) =>
+    items.filter((item) => shouldFilterItem ?
```

Comment:
Why is this check necessary?

Reply:
`shouldFilterItem` is possibly undefined and results in a ts error if we do not check it.